packages/providers/providers.yaml

```diff
@@ -7841,6 +7878,24 @@ notion:
             response_path: results
     docs: https://docs.nango.dev/integrations/all/notion
 
+notion-mcp:
+    display_name: Notion (MCP)
+    categories:
+        - knowledge-base
+        - productivity
+        - mcp
+    auth_mode: MCP
+    authorization_url: https://mcp.notion.com/authorize
+    token_url: https://mcp.notion.com/token
+    registration_url: https://mcp.notion.com/register
+    authorization_params:
+        response_type: code
+    token_params:
+        grant_type: authorization_code
+    refresh_params:
+        grant_type: refresh_token
+    docs: https://docs.nango.dev/integrations/all/notion
```

Comment:
Not necessary for now, but eventually do we want dedicated docs pages for this?

Reply:
Yes, for sure. Just didn't feel it was worth it yet adding in docs for it.